biz.aQute.bndlib/src/aQute/bnd/build/Project.java

```diff
@@ -2112,6 +2112,15 @@ public void test(File reportDir, List<String> tests) throws Exception {
 			IO.delete(reportDir);
 			tester.setReportDir(reportDir);
 		}
+		if (tests == null) {
+			// check system properties: gradle
+			String singleTest = System.getProperty("integTest.single");
```

Comment:
`tester.names` is not for us to use. It if for the project to communicate to the launched VM. 

> finally, these ARE integration tests.. that's why I applied the integration test properties defined by maven and gradle and not the "unit" test properties.

Yes, but you did not properly read the gradle description:

```
Setting a system property of taskName.single = testNamePattern will only execute tests that match the specified testNamePattern. The taskName can be a full multi-project path like “:sub1:sub2:test” or just the task name.

gradle -Dtest.single=:proj1:test:Customer build
gradle -DintegTest.single=c/d/ :proj1:integTest
```
The property name depends upon the task name. `integTest` is not some special name. It is just the name of the task in the example.